main.go

```diff
@@ -170,7 +171,7 @@ func main() {
 	err = root.Execute()
 
 	if err != nil {
-		message := err.Error()
+		message := suggest.UserFriendlyError(err).Error()
```

Comment:
not sure i agree on having this at the main, so all the error will go though the "suggestor", but not all would be...

Can we throw them from the origin? Related to frans suggestion i think we should raise a UserFriendlyError when is potentially triggered (serializer?) and then that goes around as error until is returned at the client.

Reply:
@teresaromero This is a good point of dicussion. I look forward to hear more ideas. My point of view is that the "centralized" approach is looking at errors that now we don't want to modify but if in the future we want to extend the "content" we can easily add a new rule, if we change errors at the origin we would need to modify the code in N parts, making each new addition more complicated and in some occasions very intruisive. For example we might want to log "technical" errors inside, but to the user we want to present a different error. 

I also have the feeling that having it here could have an unwanted behaviour but on the other side the benefits above make me prefer the current approach.

Reply:
After discussing offline we agreed to change this, the latest commits have this change, please have a look 🙏